### PR TITLE
Add tests for listbox/listbox-collapsible.html

### PR DESCRIPTION
**Pull request by spectranaut**

### CODE REVIEW

**Comment by mcking65** [comment]
@jessebeach, @tatermelon, @sh0ji, this is ready for review.

**Comment by mcking65** [comment]
@spectranaut, this now has conflicts.

**Comment by spectranaut** [comment]
@mcking65 -- should I switch this PR to be a PR against master, instead of the bocoup branch?

**Comment by spectranaut** [comment]
ok I fixed the merge conflicts, @mcking65, and this PR is still a PR into bocoup.

File: test/tests/listbox-collapsible.js [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+  // Check that aria-expanded is not set and the listbox is not visible before interacting
+
+  await assertAttributeDNE(t, ex.buttonSelector, 'aria-expanded');
+
+  t.false(
+    await t.context.session.findElement(By.css(ex.listboxSelector)).isDisplayed(),
+    'listbox element should not be displayed when \'aria-expanded\' does not exist'
+  );
+
+  // click button
```
Comment (jessebeach) [inline test/tests/listbox-collapsible.js]: Is this meant to be turned into an action later?

File: test/tests/listbox-collapsible.js [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+  await listbox.sendKeys(Key.ARROW_DOWN);
+
+  let newSelectedText = await t.context.session.findElement(
+    By.css(ex.optionSelector + '[aria-selected="true"]')
+  ).getText();
+
+  // Send ESCAPE to the listbox
+  await listbox.sendKeys(Key.ESCAPE);
+
+  // And focus should be on the element with the corrosponding text in on button
```
Comment (jessebeach) [inline test/tests/listbox-collapsible.js]: nit: 'corresponding'

File: test/tests/listbox-collapsible.js (hunk 1 above)
Comment (spectranaut) [reply test/tests/listbox-collapsible.js]: it just describes the next line of code :)

**Pull request merged**